Examples/src/main/kotlin/com/mapbox/vision/examples/POIActivityKt.kt

```diff
@@ -0,0 +1,153 @@
+package com.mapbox.vision.examples
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.os.Environment
+import com.mapbox.vision.VisionReplayManager
+import com.mapbox.vision.mobile.core.interfaces.VisionEventsListener
+import com.mapbox.vision.mobile.core.models.AuthorizationStatus
+import com.mapbox.vision.mobile.core.models.Camera
+import com.mapbox.vision.mobile.core.models.Country
+import com.mapbox.vision.mobile.core.models.FrameSegmentation
+import com.mapbox.vision.mobile.core.models.classification.FrameSignClassifications
+import com.mapbox.vision.mobile.core.models.detection.FrameDetections
+import com.mapbox.vision.mobile.core.models.position.VehicleState
+import com.mapbox.vision.mobile.core.models.road.RoadDescription
+import com.mapbox.vision.mobile.core.models.world.WorldDescription
+import java.io.IOException
+import java.io.InputStream
+import kotlinx.android.synthetic.main.activity_main.vision_view
+import kotlinx.android.synthetic.main.activity_poi.*
+
+class POIActivityKt : BaseActivity() {
+
+    private var visionReplayManagerWasInit = false
+
+    // Download session from tutorial and push to device
+    private var sessionPath = "${Environment.getExternalStorageDirectory().absolutePath}/session"
+
+    private val visionEventsListener = object : VisionEventsListener {
+
+        private var cameraCalibrated = false
+
+        override fun onAuthorizationStatusUpdated(authorizationStatus: AuthorizationStatus) {}
+
+        override fun onFrameSegmentationUpdated(frameSegmentation: FrameSegmentation) {}
+
+        override fun onFrameDetectionsUpdated(frameDetections: FrameDetections) {}
+
+        override fun onFrameSignClassificationsUpdated(frameSignClassifications: FrameSignClassifications) {}
+
+        override fun onRoadDescriptionUpdated(roadDescription: RoadDescription) {}
+
+        override fun onWorldDescriptionUpdated(worldDescription: WorldDescription) {}
+
+        override fun onVehicleStateUpdated(vehicleState: VehicleState) {
+            if (cameraCalibrated) {
+                poi_view.updatePOIState(vehicleState.geoLocation)
+            }
+        }
+
+        override fun onCameraUpdated(camera: Camera) {
+            if (camera.calibrationProgress == 1.0f) {
```

Comment:
nit: don't we want to show current calibration while camera is calibrating?

Reply:
added calibration progress